src/scripts/install.sh

```diff
@@ -123,21 +130,21 @@ Uninstall_AWS_CLI() {
 }
 
 if [ ! "$(command -v aws)" ]; then
-    if [ "$ORB_VAL_AWS_CLI_VERSION" = "latest" ]; then
-        Install_AWS_CLI
+    if [ "$ORB_STR_AWS_CLI_VERSION" = "latest" ]; then
+        Install_AWS_CLI ""
```

Comment:
Please double-check if this line is supposed to have the `""` at the end.
```suggestion
        Install_AWS_CLI
```

Reply:
In this particular issue, the user set the shell to `shell: /bin/bash -o nounset`. This means that if there's a variable that's unbounded in a script, it will automatically fail.

When calling the  `Install_AWS_CLI` function, the job fails because `$1` is unbounded. 

In the `Install_AWS_CLI` function, `$1` is the version argument passed into the function. For example, If I passed `v2.1.0` into ` Install_AWS_CLI v2.1.0`, `$1 = v2.1.0`.

If nothing is passed into  `Install_AWS_CLI`, the latest version will be downloaded but it will fail with a shell that's set to `nounset`. 

To get around this issue, I have to pass in an empty string as an argument when calling ` Install_AWS_CLI` so that `$1 = ""` and is not unbounded.